docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-ha-proc.adoc

```diff
@@ -0,0 +1,42 @@
+[id='environment-authoring-ha-proc']
+= Deploying a high-availability authoring environment
+
+To deploy a high-availability authoring environment, use the `rhpam-70-authoring-ha.yaml` template file. You can download this file from `{URL_BASE_GITHUB_PAM}/templates`.
+
+.Procedure
+. Use one of the following methods to deploy the template:
+.. In the OpenShift Web UI, select *Add to Project >  Import YAML / JSON* and then select or paste the `rhpam-70-authoring-ha.yaml` file. In the *Add Template* window, ensure *Process the template* is selected and click *Continue*.
+.. To use the OpenShift command line console, prepare the following command line:
++
+[subs="verbatim,macros"]
+----
+oc new-app -f <template-path>/rhpam-70-authoring-ha.yaml -p PARAMETER=value -p PARAMETER=value...
+----
++
+In this command line:
++
+* Replace `<project-name>` with the name of your project
+* Replace `<template-path>` with the path to the downloaded template file.
+* Use as many `ATTRIBUTE=value` pairs as needed to set the required attributes. You can view the template file to see descriptions for the attributes.
++
```

Comment:
Formatting gets off starting at step 2 (see output). Play with the + line break and see what it needs.

Reply:
I don't see it off
![image](https://user-images.githubusercontent.com/33361640/40235899-98fb0ff6-5aa3-11e8-8fe5-1151af687e26.png)

Reply:
My bad. Looked at it wrong. The double levels before it threw my eye off. All good.